docs/source/getting-started.rst

```diff
@@ -326,4 +326,179 @@ Thats it, you are now serving a Surround pipeline! Now you could potentially use
 application.
 
 .. note:: Since this project was generated with a web runner, the default mode is ``web``, to run the pipeline
-        using the ``FileSystemRunner`` instead, use the command ``$ surround run batch`` or ``$ surround run train``.
\ No newline at end of file
+        using the ``FileSystemRunner`` instead, use the command ``$ surround run batch`` or ``$ surround run train``.
+
+Serving your first pipeline via Google Cloud Compute
+********************************************
+
+**Introduction**
```

Comment:
Use headings instead of just making them bold
e.g. 
```
Introduction
^^^^^^^^^^^^
```

Reply:
I've fixed that ! Thanks Zac